<!-- -->
<launch>
    <node pkg="fix2pose" type="fix2tfpose" name="sn_pose" ns="/sn" output="screen">
        <param name="topic_pose" value="/sn"/>
		<param name="topic_gps" value="/sn/fix"/>
        <param name="frame_id" value="/sn"/>
        <param name="topic_yaw_gps" value="/sn/raw_data"/>
		<param name="topic_yaw" value="/sn/heading"/>
		<param name="origin_lat" value="31.027"/>
		<param name="origin_lon" value="121.433"/>
    </node>
    
    <node pkg="fix2pose" type="pose2path" name="sn_path" output="screen">
        <param name="pose_name" type="string" value="/sn"/>
        <param name="path_name" type="string" value="/sn_path"/>
        <param name="gps_name" type="string" value="/sn/fix"/>
        <param name="gps_raw_name" type="string" value="/sn/raw_data"/>
        <param name="point_marker_type" type="int" value="6"/>
        <param name="id" type="int" value="100"/>
        <param name="display_type" type="int" value="2"/>
    </node>
    <node pkg="fix2pose" type="pose2path" name="sn_markers" output="screen">
        <param name="pose_name" type="string" value="/sn"/>
        <param name="path_name" type="string" value="/sn_markers"/>
        <param name="gps_name" type="string" value="/sn/fix"/>
        <param name="gps_raw_name" type="string" value="/sn/raw_data"/>
        <param name="point_marker_type" type="int" value="6"/>
        <param name="marker_size" type="double" value="0.2"/>
        <param name="id" type="int" value="101"/>
        <param name="display_type" type="int" value="0"/>
    </node>

</launch>
